bedrock/privacy/templates/privacy/index.html

```diff
@@ -83,7 +83,13 @@ <h1 itemprop="name">{{ doc.h1.string }}</h1>
         <li class="policy-firefox-os"><a href="{{ url('privacy.notices.firefox-os') }}">{{ _('Firefox OS') }}</a></li>
         <li class="policy-firefox-cloud"><a href="{{ url('privacy.notices.firefox-cloud') }}">{{ _('Firefox Cloud Services') }}</a></li>
         <li class="policy-marketplace"><a href="https://marketplace.firefox.com/privacy-policy">{{ _('Firefox Marketplace') }}</a></li>
-        <li class="policy-firefox-focus"><a href="{{ url('privacy.notices.firefox-focus') }}">{{ _('Firefox Focus') }}</a></li>
+        <li class="policy-firefox-focus">
+        {% if LANG == 'de' %}
+          <a href="{{ url('privacy.notices.firefox-klar') }}">Firefox Klar</a>
```

Comment:
We don't usually expose brand names to locales. In this case, it would also be a waste of resources, since all locales would have to confirm it, with the exception of German.